torch/csrc/jit/script/sugared_value.cpp

```diff
@@ -29,6 +29,42 @@ std::shared_ptr<SugaredValue> PrintValue::call(
   return std::make_shared<NoneValue>();
 }
 
+std::shared_ptr<SugaredValue> SortedValue::call(
+    const SourceRange& loc,
+    Function& m,
+    at::ArrayRef<NamedValue> inputs,
+    at::ArrayRef<NamedValue> attributes,
+    size_t n_binders) {
+  auto& g = *m.graph();
+  if (!attributes.empty())
+    throw ErrorReport(loc) << "sorted doesn't accept any keyword arguments";
+
+  if (inputs.size() != 1)
+    throw ErrorReport(loc)
+        << "sorted currently only accept any keyword arguments";
```

Comment:
`"only accepts 1 positional argument"`

Reply:
i didn't want to repeat all the code & ad-hoc schema checking that exists for `aten::sort`. i think it is more maintanable to compose this in terms of `aten::sort`